clone-awx-operator.py

```diff
@@ -14,9 +14,22 @@
 import sys
 import tempfile
 
-DEFAULT_BRANCH = "devel"
 DEFAULT_AWX_OPERATOR_REPO = "https://github.com/ansible/awx-operator"
 
+# get the appVersion configured in Chart.yaml
+def get_app_version():
+    # naive implementation without yaml-parsing package
```

Comment:
I don't mind adding a dependency and calling `yaml.safe_load()`. It might be a little more robust.

Reply:
I wasn't sure whether we wanted to open the can of worms from the python standpoint, given all of the options around dependency management (venv, Dockerfile, etc)

Reply:
I'm happy to leave it to your preference. Maybe the fewer lines of code isn't worth the overhead of adding requirements files and all that.